### ISSUE TITLE
Set node set name for multiple node sets for different graph tensors

### ISSUE THREAD

**Issue by SidneyLann** [joshcarty/tfgnn-ogb#11]
hidden = gnn.Convolve()(input)
    for _ in range(num_neighbourhoods - 1):
        hidden = gnn.Convolve()(hidden)
    hidden = tfgnn.keras.layers.ReadoutFirstNode(node_set_name="paper")(hidden)
    output = tf.keras.layers.Dense(num_classes, activation="softmax")(hidden)
    return model.NodeClassificationModel(
        input, output, target_node="paper", label_name="label"
    )

For above example, there is only one node set, If there were multiple node sets, how to pass the target_node argument for different graph tensors then?

**Comment by joshcarty** [comment]
Thanks, that's great to hear the repository has been useful to you.

I created this example in the absence of examples in [tensorflow/gnn](https://github.com/tensorflow/gnn). There are now official examples so I won't be maintaining or extending this repository.

I hope you understand. Thanks!
